selinux/ceph.te

```diff
@@ -86,6 +88,13 @@ logging_send_syslog_msg(ceph_t)
 
 sysnet_dns_name_resolve(ceph_t)
 
+corenet_tcp_connect_afs3_callback_port(ceph_t)
```

Comment:
i am suspicious about these ports: 6969,7000,7001,7002. how come ceph-osd are trying to connect to these ports be default?